Subject: declare cpu_clock as extern function
Add extern declaration for the sched.c cpu_clock function to
resolve the implicit declaration in softlockup.c.

signed-off-by: Sven-Thorsten Dietrich <sdietrich@novell.com>


---
 include/linux/sched.h |    1 +
 1 files changed, 1 insertion(+)

--- linux-2.6.22.orig/include/linux/sched.h
+++ linux-2.6.22/include/linux/sched.h
@@ -307,6 +307,7 @@ extern void cpu_init (void);
 extern void trap_init(void);
 extern void update_process_times(int user);
 extern void scheduler_tick(void);
+extern unsigned long long cpu_clock(int cpu);
 
 #ifdef CONFIG_GENERIC_HARDIRQS
 extern int debug_direct_keyboard;
